cli/cmd/summary.go

```diff
@@ -62,3 +73,100 @@ func checkFilePath(filename string) error {
 	dirPath := filepath.Dir(filename)
 	return os.MkdirAll(dirPath, 0755)
 }
+
+func generateSummaryTable() (*schema.Table, error) {
+	tableName := "cloudquery_sync_summaries"
+	t := schema.Tables{{
+		Name: tableName,
+		Transform: transformers.TransformWithStruct(
+			&syncSummary{},
+		),
+		Columns: []schema.Column{},
+	}}
+	if err := transformers.TransformTables(t); err != nil {
+		return nil, err
+	}
+	return t[0], nil
+}
+
+func migrateSummaryTable(writeClient plugin.Plugin_WriteClient, destTransformer *transformer.RecordTransformer, spec specs.Destination) error {
+	summaryTable, err := generateSummaryTable()
+	if err != nil {
+		return err
+	}
+	summaryTableSchema := summaryTable.ToArrowSchema()
+	transformedSchema := destTransformer.TransformSchema(summaryTableSchema)
```

Comment:
Using `destTransformer` here generates `_cq_sync_time` and `_cq_source_name` columns 
![image](https://github.com/cloudquery/cloudquery/assets/26760571/bc994c15-920a-4eda-9ecb-3b5c6040e829)

Reply:
Probably will generate `_cq_sync_group_id` is enabled by the config

Reply:
I think that is a good thing, to keep consistency with all of our other tables